include/conio.h

```diff
@@ -837,4 +837,4 @@ void flushkeybuf(void);
 unsigned char cinput(
     unsigned char* buffer, unsigned char buflen, unsigned char flags);
 
-#endif // M65LIBC_CONIO_H
+#endif /* __MEGA65_CONIO_H */

```

Comment:
On this one `/* .. */` is used, but the rest in the PR uses `//`. It would hurt if it was consistent, using `//` is C99 and will not work with C89/C94. C99 is close to 25 years old, but I do not know if we expect issues with old compilers?

Reply:
Ahh ok, I think we can resolve it in a separate issue, either now or when a new compiler becomes relevant. I'm unsure of what standards are out there.